Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayAuthorizationDelegate.m

```diff
@@ -264,14 +216,14 @@ - (void)getShippingRates:(BUYCheckout *)checkout completion:(void (^)(PKPaymentA
 			if ([shippingMethods count] > 0) {
 				[self selectShippingMethod:shippingMethods[0] completion:^(BUYCheckout *checkout, NSError *error) {
 					if (checkout && error == nil) {
-						self.checkout = checkout;
+						_checkout = checkout;
```

Comment:
If we're going to standardize on using accessors for reading properties, we should use accessors for writing them. We should define private readwrite property for `checkout`.